x/cork/keeper/keeper.go

```diff
@@ -111,6 +113,141 @@ func (k Keeper) IterateCorks(ctx sdk.Context, handler func(val sdk.ValAddress, c
 	}
 }
 
+/////////////////////
+// Scheduled Corks //
+/////////////////////
+
+func (k Keeper) SetScheduledCork(ctx sdk.Context, val sdk.ValAddress, blockHeight uint64, cork types.Cork) {
```

Comment:
Nit: generally the pattern is to order the arguments in keyspace order, so we should consider adjusting that.

Reply:
Fixed